### ISSUE TITLE
AppNavigation is still traversable when closed

### ISSUE THREAD

**Issue by danxuliu** [nextcloud/nextcloud-vue#2982]
When [the `AppNavigation` is closed it is moved outside of the viewport](https://github.com/nextcloud/nextcloud-vue/blob/5213d5f1370e54de7bf684c4df0f2d510e72c493/src/components/AppNavigation/AppNavigation.vue#L165-L168). However, its contents are still in the DOM and, technically, visible, so they are still focusable and pressing tab traverses them.

Note that the `AppNavigation` as a whole can not be hidden when closed, as [the toggle to show it again is also part of it](https://github.com/nextcloud/nextcloud-vue/blob/5213d5f1370e54de7bf684c4df0f2d510e72c493/src/components/AppNavigation/AppNavigation.vue#L61); only the list and the footer should be hidden.

## Steps to reproduce

- Open Talk or Mail
- Close the app navigation
- Traverse the UI pressing tab

### Expected result

The button to toggle the navigation bar is traversed, but nothing else from the AppNavigation is

### Actual result

The contents of the navigation bar are traversed
